community/kernel/src/main/java/org/neo4j/helpers/Settings.java

```diff
@@ -1,943 +0,0 @@
-/*
- * Copyright (c) 2002-2016 "Neo Technology,"
- * Network Engine for Objects in Lund AB [http://neotechnology.com]
- *
- * This file is part of Neo4j.
- *
- * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package org.neo4j.helpers;
-
-import java.io.File;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.EnumSet;
-import java.util.List;
-import java.util.function.BiFunction;
-import java.util.function.Function;
-import java.util.regex.Pattern;
-
-import org.neo4j.function.Functions;
-import org.neo4j.graphdb.config.InvalidSettingException;
-import org.neo4j.graphdb.config.Setting;
-import org.neo4j.helpers.collection.Iterables;
-import org.neo4j.io.fs.FileUtils;
-import org.neo4j.kernel.configuration.Config;
-
-/**
- * Create settings for configurations in Neo4j. See {@link org.neo4j.graphdb.factory.GraphDatabaseSettings} for example.
- *
- * Each setting has a name, a parser that converts a string to the type of the setting, a default value,
- * an optional inherited setting, and optional value converters.
- *
- * A parser is a function that takes a string and converts to some type T. The parser may throw IllegalArgumentException
- * if it fails.
- *
- * The default value is the string representation of what you want as default. Special values are the constants NO_DEFAULT,
- * which means that you don't want any default value at all, and MANDATORY, which means that the user has to specify a value
- * for this setting. Not providing a mandatory value for a setting leads to an IllegalArgumentException.
- *
- * @deprecated this class is deprecated and will be moved to internal packages in the next major release
- */
-@Deprecated
```

Comment:
Unfortunately, this was not deprecated in 2.x :( https://github.com/neo4j/neo4j/commit/1ba7202f896b04d53bb14a68972dca170895617c